### PR TITLE
Fix issue with images not rendered inside a tabnav, in Safari

### PR DESCRIPTION
**Pull request by dobromir-hristov**
Bug/issue #, if applicable: 105363672

## Summary

This fixes a bug where images are not rendered inside tabnavs, for Safari only.

This only happens if the image is a `display: block`, and has some element before it. Really strange, but removing the `loading: lazy` fixes it.

## Dependencies

NA

## Testing

Compile and try with this:

```md
@TabNavigator{
  @Tab("Image With Title"){
    **Strong Title**

    @Image(source: example.png)
  }
  @Tab("Image with Caption"){
    @Image(source: example.png){ Some text as a placeholder }
  }
}
```

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [x] Added tests
- [x] Ran `npm test`, and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

**Comment by mportiz08** [comment]
I think we could potentially take these changes as a workaround, but I'm wondering if there's a simpler problem happening here.

It looks like the `overflow: hidden` rule from the `.tabs-content-container` element is the main thing preventing the image from being loaded. If I uncomment that one style, the image loads as expected—since it is technically overflowing, it seems like Safari is the only browser considering the image as not being visible and thus not loading it since we're using the lazy loading strategy.

Can you remind me if that rule is strictly necessary and what scenario it prevents? I'm sure there is one, just curious if it might be better to consider a solution at the CSS level for this or not.

**Comment by dobromir-hristov** [comment]
When switching between tabs, there is a slight transition. 

When sections have different heights, for example you go from a long section to a short section, the `overflow` rule ensures the long section does not spill out into the next elements, below the tabnav. 

This would not be an issue if there is no animation at all, but since for a brief moment both sections are visible at the same time, one is absolutely positioned while the other takes the space it needs.

File: src/components/ContentNode/TabNavigator.vue [hunk 1]
```diff
@@ -33,6 +33,7 @@
 <script>
 import Tabnav from 'docc-render/components/Tabnav.vue';
 import TabnavItem from 'docc-render/components/TabnavItem.vue';
+import ImageLoadingStrategy from '@/constants/ImageLoadingStrategy';
```
Comment (mportiz08) [inline src/components/ContentNode/TabNavigator.vue:36]:
```suggestion
import ImageLoadingStrategy from 'docc-render/constants/ImageLoadingStrategy';
```

File: tests/unit/components/ContentNode/TabNavigator.spec.js [hunk 2]
```diff
@@ -6,13 +6,14 @@
  *
  * See https://swift.org/LICENSE.txt for license information
  * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
-*/
+ */
```
Comment (mportiz08) [inline tests/unit/components/ContentNode/TabNavigator.spec.js:9]:
```suggestion
*/
```

**Review by mportiz08 — COMMENTED** [review]
Seems to be working well.

File: tests/unit/components/ImageAsset.spec.js [hunk 3]
```diff
@@ -6,12 +6,13 @@
  *
  * See https://swift.org/LICENSE.txt for license information
  * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
-*/
+ */
```
Comment (dobromir-hristov) [inline tests/unit/components/ImageAsset.spec.js:9]:
```suggestion
*/
```

**Comment by dobromir-hristov** [comment]
@swift-ci test

**Review by mportiz08 — APPROVED** [review]

**Comment by mportiz08** [comment]
@swift-ci test

**Pull request merged**